LibrariesPrototype/Presentation/Extensions/LTViewModel+WhisperMessage.swift

```diff
@@ -0,0 +1,82 @@
+
+//
+//  LTViewModel+WhisperMessage.swift
+//  LibrariesPrototype
+//
+//  Created by Ozgun Emre Zor on 23/04/2019.
+//  Copyright © 2019 Ozgun Zor. All rights reserved.
+//
+
+import Foundation
+import Whisper
+
+enum WhisperMessageType {
+    case whisper
+    case shout
+    case whistle
+}
+
+extension LTViewModel {
+
+    func showMessage(withType type: WhisperMessageType) {
+        switch type {
+        case .whisper:
+            showWhisper()
+        case .shout:
+            showShout()
+        case .whistle:
+            showWhistle()
+        }
+    }
+
+    private func showWhisper() {
+        let message = Message(title: "Enter your message here.", backgroundColor: .red)
+
+
+        guard let viewController = Session.shared.currentViewController else {
+            return
+        }
+
+        // Show and hide a message after delay
+//        Whisper.show(whisper: message, to: navigationController, action: .show)
+
+        guard let navigationController = viewController.navigationController else {
+            return
+        }
+
+        // Present a permanent message
+        Whisper.show(whisper: message, to: navigationController)
+
+        // Hide a message
+        Whisper.hide(whisperFrom: navigationController)
+    }
+
+    private func showShout() {
+
+        guard let viewController = Session.shared.currentViewController else {
+            return
+        }
+
+        guard let navigationController = viewController.navigationController else {
```

Comment:
+inline

Reply:
You mean like this right
```suggestion
        guard let navigationController = viewController.navigationController else { return }
```